operator/internal/manifests/internal/alerts/prometheus-alerts.yaml

```
@@ -209,3 +209,21 @@ groups:
     for: 1m
     labels:
       severity: warning
+  - alert: LokiIngesterCriticalFlushFailureRate
+    annotations:
+      summary: "Loki ingester has critical flush failure rate"
+      description: |
+        Loki ingester {{ $labels.pod }} has a critical flush failure rate of {{ $value | humanizePercentage }}
+        over the last 5 minutes. This requires immediate attention as data is not being flushed to the storage.
+        Validate if the storage configuration is still valid and if the storage is still reachable.
+        Current failure rate: {{ $value | humanizePercentage }}
+        Threshold: 20%
+    expr: |
+      (
+        rate(loki_ingester_chunks_flush_failures_total[5m])
+        /
+        rate(loki_ingester_chunks_flush_requests_total[5m])
+      ) > 0.20
```

I don't know if we want this alert per-pod or per-lokistack, but either way I think we should add an aggregation to the labels that we use, for example:


```suggestion
      sum (
        rate(loki_ingester_chunks_flush_failures_total[5m])
        /
        rate(loki_ingester_chunks_flush_requests_total[5m])
      ) by (namespace, pod)
      > 0.20
```

(I have added `namespace`, but otherwise this is the aggregation "per pod" as suggested by the description)